universe/vault/types.proto

```diff
@@ -9,19 +9,20 @@ package foundation.universe.vault;
 option go_package = "namespacelabs.dev/foundation/universe/vault";
 
 message AppRole {
-    string        name       = 1;
-    string        auth_mount = 2;
-    VaultProvider provider   = 3;
+    string name  = 1;
+    string mount = 3;
```

Comment:
The other way around, mount = 2, reserved 3, no?

Reply:
fixed